tests/shopify_python/test_git_utils.py

```diff
@@ -248,3 +248,71 @@ def test_autopep_files(tmpdir):
     for fixed_file in files_to_autopep:
         with open(fixed_file) as file_to_check:
             assert file_to_check.readlines() == file_lines
+
+
+def test_linter(tmpdir):
+    # type: ('py.path.LocalPath') -> None
+    open(os.path.join(str(tmpdir), '__init__.py'), 'w')
+
+    file_lines = [
+        "def foo():\n",
+        "  return 1\n"
+    ]
```

Comment:
I tried this, but it actually became harder to figure out due to the need to also end in a `\n` that the `strip()` call removed. I like it better as-is.